app/_how-tos/dynamically-rewrite-simple-request-urls-with-routes.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Dynamically rewrite simple request URLs with {{site.base_gateway}} Routes
+content_type: how_to
+related_resources:
+  - text: Routes
+    url: /gateway/entities/route/
+  - text: Services
+    url: /gateway/entities/service/
+  - text: Routing in {{site.base_gateway}}
+    url: /gateway/routing/
+  - text: Expressions router
+    url: /gateway/routing/expressions/
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tools:
+    - deck
+
+prereqs:
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+entities:
+  - service
+  - route
+
+tags:
+  - routing
+  - traffic-control
+
+tldr:
+  q: How can I divert traffic from an old URL to a new one with {{site.base_gateway}}?
+  a: Set up a Gateway Service with the old URL path and create a new a Route with new path.For example, your legacy upstream endpoint may have a base URI like `/api/old/`. However, you want your publicly accessible API endpoint to now be named `/new/api`. To route the Service’s upstream endpoint to the new URL, you can set up a Service with the path `/api/old/` and a Route with the path `/new/api`.
```

Comment:
I thinjk the /api/new and /api/old are good examples of what you are trying to do but because the deck file and steps dont reflect that I got thrown off. I think that the api/old and api/new can be illustrated through the steps better than as the answer int he tld;r